### ISSUE TITLE
Explain within client what each accuracy level means

### ISSUE THREAD

**Issue by stefanb** [traccar/traccar-client-android#381]
Currently users cannot know what it means and it seems that changing accuracy to High solves significant portion of support requests.

My first interpretation of the "Location accuracy" setting was how many decimals should be included in the sent coordinates ;-) but in fact it affects also other reporting parameters (speed, direction...)

**Comment by reslin** [comment]
stefanb maybe intended to ask that the Traccar-Frontend (which happens to be a PreferenceFragment) should be (or might be) more informative:

- tell the meaning of an item
- tell the status/content of an item

**Comment by stefanb** [comment]
Yes, user has no ide

**Comment by Tyler-2** [comment]
Is it accurate (hehe) to say that when Medium accuracy is set, that GPS High accuracy would be used if another app is using GPS? I notice LOW depends on the last acquired position of any App on your phone, I wonder if MEDIUM can also make use of other app's GPS requests?

TLDR: Does a phone running a map/driving app like Google Maps then allow a Traccar Client set to Medium to make use of the high precision GPS fix that the other app is using?

**Comment by tananaev** [comment]
I don't think so, but Google doesn't really disclose those details in the documentation.
